src/core_plugins/kibana/public/visualize/listing/visualize_listing.html

```diff
@@ -39,7 +39,7 @@
           ng-click="listingController.deleteSelectedItems()"
           tooltip="Delete selected visualizations"
         >
-          <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-trash"></span>
+          <trash-icon-component></trash-icon-component>
```

Comment:
Personally, I don't think there is any reason to indicate that these angular directives are backed by react components. When the react components were sitting in the angular $injector, it was a different story, but now I think it makes more sense to hide that detail and just treat them like every other directive